server/src/test/java/org/elasticsearch/client/documentation/IndicesDocumentationIT.java

```diff
@@ -89,6 +93,11 @@ public void testPutMappingDocumentation() throws Exception {
                 "}", XContentType.JSON)
         .get();
         // end::putMapping-request-source
+        getMappingsResponse = client.admin().indices().prepareGetMappings("twitter").get();
+        assertEquals(1, getMappingsResponse.getMappings().size());
+        indexMapping = getMappingsResponse.getMappings().get("twitter");
+        assertThat(indexMapping.get("user"), instanceOf(MappingMetaData.class));
+        assertThat(indexMapping.get("user").getSourceAsMap().toString(), equalTo("{properties={name={type=text}}}"));
```

Comment:
small thing, but the `toString` comparison feels a bit brittle here: whitespace and other kind of cosmetic changes can break this prettty easily. could we instead assert on the Map using `containsKey` or similar?

Reply:
Yes, I was thinking about this but wanted to avoid few extra lines of code here, since I found they would also make the test harder to read. I generally agree with "toString" being brittle. In this case, since most maps use AbstractMap#toString(), it should be quite stable though. So maybe okay to trade for succinctness and readability here, wdyt?

Reply:
I'm good with your reasoning.

Reply:
Actually, I found a way that avoids "toString()" and is quite short as well, please take a look at the next commit to see if you like this better.